x-pack/solutions/security/plugins/security_solution/public/management/pages/policy/view/policy_settings_form/components/device_control_notify_user_option.tsx

```diff
@@ -51,6 +52,10 @@ export const DeviceControlNotifyUserOption = React.memo(
       return policy.windows.device_control?.enabled || policy.mac.device_control?.enabled || false;
     }, [policy]);
 
+    const currentAccessLevel = useMemo(() => {
+      return policy.windows.device_control?.usb_storage || policy.mac.device_control?.usb_storage;
+    }, [policy]);
```

Comment:
nit: `useMemo` is most probably unnecessary here

Reply:
You are right, removed - 9c0c8323f5cf5372f873c0290a8078679d91e7fc